src/models/Model.ts

```diff
@@ -117,17 +144,41 @@ export class Model extends BaseComponent {
     }
   }
 
+  /**
+   * Sets a new default value to a single specific attribute.</br>
+   * Note: specifying a new attribute default value does not set the attribute to that value. This can be done using the {@link setDefault} method.
+   * @param attribute
+   * the specific attribute whose default value is to be changed.
+   * @param value
+   * the new intended default value.
+   * @param options
+   * if the `customAttribute` option is set to `true`, the method will not validate whether the attribute is registered or not.
+   */
   public setNewDefault(attribute: string, value: any, options?: IModelSetOptions) {
     if (!options || !options.customAttribute) {
       this.checkIfAttributeExists(attribute);
     }
     this.defaultAttributes[attribute] = value;
   }
 
-  public setDefault(attribute: string, options?: IModelSetOptions) {
+  // I removed the unused 'options' parameter in the setDefault method. FB
```

Comment:
Remove this comment :)

If you wish to inform us about why you are doing such a change, do it in the code review